connectors/src/api/webhooks/webhook_google_drive.ts

```diff
@@ -56,21 +56,21 @@ const _webhookGoogleDriveAPIHandler = async (
   }
   const connector = await ConnectorResource.fetchById(connectorId);
   if (!connector) {
```

Comment:
I guess in this state we can only have a webhook corresponding to a deleted connector. Am I right, here?

Reply:
I agree with your analysis :+1: